### PR TITLE
twindle dark theme

### PR DESCRIPTION
**Pull request by frankiefab100**
## Description
Twindle dark theme

### CODE REVIEW

**Comment by Akshay2996** [comment]
Hi @frankiefab100 Can we get a Screenshot of what it looks like & no need to write so much CSS in dark-theme.css

You only have to write the CSS variables that needs to be changed i.e **only :root should remain & delete everything else in the CSS** Everything will work fine. Try this yourself first.

File: misc/twindle-web/dark-theme.css [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+}
+
+.sticky {
+  box-shadow: 0 2px 4px #353632;
+}
+
+.nav-links,
+.logo {
+  color: #fff;
+}
```
Comment (Akshay2996) [inline misc/twindle-web/dark-theme.css:38]: Please can you fix this. Create a CSS variable & assign this color value to it. There are same problem in the whole file

File: misc/twindle-web/dark-theme.css [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+.main-nav li a {
+  color: #fff;
+}
```
Comment (Akshay2996) [inline misc/twindle-web/dark-theme.css:42]: Here too.

File: misc/twindle-web/dark-theme.css [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+.logo {
+  color: #dbdbdb;
+}
+
```
Comment (Akshay2996) [inline misc/twindle-web/dark-theme.css:47]: Here too. Please see where there is color define, we can use CSS variable instead to minimize rewriting the code

**Review by Akshay2996 — CHANGES_REQUESTED** [review]

**Comment by frankiefab100** [comment]
Okay, I will resolve this. Please give me some time.

**Comment by frankiefab100** [comment]
## This is the screenshot

Twindle_Create_books_out_of_your_tweets
Twindle_2

**Pull request merged**